src/main/java/seedu/recipe/storage/JsonAdaptedRecipe.java

```diff
@@ -19,61 +20,47 @@
 class JsonAdaptedRecipe {
 
     public static final String MISSING_FIELD_MESSAGE_FORMAT = "Recipe's %s field is missing!";
+    private String name;
+    private String portion;
+    private String duration;
+    private String tags;
```

Comment:
Might be appropriate to store these fields as per the old method, using a list of `JsonAdaptedXXX`